modules/default/calendar/calendar.js

```diff
@@ -131,6 +134,14 @@ Module.register("calendar", {
 			// fetcher till cycle
 			this.addCalendar(calendar.url, calendar.auth, calendarConfig);
 		});
+
+		// Refresh the DOM every minute if needed: When using relative date format for events that start
+		// or end in less than an hour, the date shows minute granularity and we want to keep that accurate.
+		setTimeout(() => {
```

Comment:
why is that setTimeout wrapped around setInterval?

Reply:
The idea is to trigger the calendar refresh on the minute change such that the time is always accurate. If you just use an interval, you will off by half a minute on average :)